lib/src/components/month_view_components.dart

```diff
@@ -178,6 +182,6 @@ class MonthPageHeader extends CalendarPageHeader {
           onTitleTapped: onTitleTapped,
           dateStringBuilder: MonthPageHeader._monthStringBuilder,
         );
-  static String _monthStringBuilder(DateTime date) =>
+  static String _monthStringBuilder(DateTime date, [DateTime? secondaryDate]) =>
```

Comment:
Can we use named parameter for `secondaryDate`?
Ex : `static String _dayStringBuilder(DateTime date, {DateTime? secondaryDate})`